### PR TITLE
async fang

### PR DESCRIPTION
**Pull request by ayrat555**

### CODE REVIEW

File: _posts/2022-08-08-async-processing.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+---
+title: Fang: Async background processing for rust
```
Comment (pxp9) [inline _posts/2022-08-08-async-processing.md]: Maybe it is better `Fang , Async ...` , also for some reason it is failing in hugo yaml

File: _posts/2022-08-08-async-processing.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+```rust
+let task = MyTask::new(0);
+
+queue
+    .insert_task(&task1 as &dyn AsyncRunnable)
+    .await
+    .unwrap();
+```
+
+Pitfalls
```
Comment (pxp9) [inline _posts/2022-08-08-async-processing.md:148]: `##` is not necessary ??

File: _posts/2022-08-08-async-processing.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+- async traits
+- heavy tasks
+  separate runtime or threaded processing
```
Comment (pxp9) [inline _posts/2022-08-08-async-processing.md:152]: `-` also needed i think

File: _posts/2022-08-08-async-processing.md (hunk 1 above)
Comment (ayrat555) [reply _posts/2022-08-08-async-processing.md]: done

**Pull request merged**